plans/bitswap-tuning/test/transfer.go

```diff
@@ -170,11 +185,12 @@ func Transfer(runenv *runtime.RunEnv) {
 	runenv.OK()
 }
 
-func setupSeed(ctx context.Context, node *utils.Node, fileSize int) (cid.Cid, error) {
+func setupSeed(ctx context.Context, node *utils.Node, fileSize int, runenv *runtime.RunEnv) (cid.Cid, error) {
```

Comment:
Do we intend to use the `runenv` in this function going forward?

Reply:
Good catch, I'll remove in the next PR I do

Reply:
Addressed in https://github.com/ipfs/testground/pull/353